src/main/resources/application.properties

```diff
@@ -18,5 +18,11 @@ mutation.data.file.prefix=data_
 mutation.metadata.file=meta_mutation.txt
 mutation.default.merged.maf.file=data_extended.maf
 
+#################################### Genome Nexus ###################################################
```

Comment:
Remove this

Reply:
Are you talking about genome nexus or mutation config ? We need both right..

Reply:
It shouldn't be in this PR

Reply:
@DixitPatel: I think what @zheins is saying that this PR is not about variant annotation, but fixes to file extraction / compression.  Can you move these properties to PR #12?